src/Chainweb/Version.hs

```diff
@@ -755,12 +738,6 @@ oldTargetGuard _ _ = False
 -- live chains, enforcing the following condition must be ignored for the
 -- historical blocks for which both the Nonce and Flags could be anything.
 --
-skipFeatureFlagValidationGuard
-    :: ChainwebVersion
-    -> BlockHeight
-        -- ^ height of header
-    -> Bool
+skipFeatureFlagValidationGuard :: ChainwebVersion -> BlockHeight -> Bool
 skipFeatureFlagValidationGuard Mainnet01 h = h < 530500  -- ~ 2020-05-01T00:00:xxZ
-skipFeatureFlagValidationGuard Development _ = True
```

Comment:
This change brings devnet in line with mainnet behavior > 530500

Reply:
The current devnet uses `h < 140` for testing the fork behavior. Those tests are done, so we can drop this.